app/services/Controller/AssetsService.php

```diff
@@ -77,7 +78,9 @@ private function setJsCollection()
                 ->setTargetUri('assets/global.js')
                 ->addJs($this->getPath('public') . 'js/jquery-3.2.1.min.js', true, false)
                 ->addJs($this->getPath('public') . 'js/bootstrap.min.js', true, false)
-                ->addJs($this->getPath('public') . 'js/editor.min.js', true, false)
+                ->addJs($this->getPagedownPath('Markdown.Converter.js'), true)
+                ->addJs($this->getPagedownPath('Markdown.Sanitizer.js'), true)
+                ->addJs($this->getPagedownPath('Markdown.Editor.js'), true)
```

Comment:
@sergeysviridenko Can we use a separate collection for the editor? I would like have an ability to enable this collection only if needed arises.